src/scripts/main.js

```diff
@@ -0,0 +1 @@
+console.log('pesho');
```

Comment:
Yes it is, However I will throw in a hello world example just for now so the JS spits out anything and if an error occurs we know about it